### ISSUE TITLE
Map dissapears/tiles don't load fully after zooming on Android 4.2.2 using Chrome 

### ISSUE THREAD

**Issue by JohnCGH** [Leaflet/Leaflet#4384]
Even navigating to the main leafletjs.com, the sample map is visible when the page loads. But as soon as I zoom in or out, when I remove my fingers from the screen, the map disappears. It seems like it is not auto-loading the new tiles at the new zoom level.

Additional Note: My own app allows for turning on and off satellite view. And when this same problem happens in my app, if I toggle to sat view, then toggle back to normal map view, the map will then load in all the tiles at the new zoom level. But if I then zoom again, the tiles will not load again and I have to toggle sat view on/off again to see the map at the new zoom level.

Basically, it seems that after doing a zoom using Chrome in Android, it doesn't reliably load in the tiles for the new zoom level.

BTW, the version of Chrome I am using is 25.0.1364.169

**Comment by yohanboniface** [comment]
> BTW, the version of Chrome I am using is 25.0.1364.169

Is that a really really old version of Chrome or am I missing something?

**Comment by IvanSanchez** [comment]
@JohnCGH Is this still an issue? I'm considering closing this due to lack of information/reproducibility.

**Comment by ormajollila** [comment]
@IvanSanchez I stumbled into this problem on my android 6.0 phone. When I zoom in and out aggressively tiles disappear. When I look at the inspector I see urls with incorrect (I believe) z values. I'm using 0.7.7.
![screen shot 2016-09-14 at 18 51 25](https://cloud.githubusercontent.com/assets/9079351/18519500/d7f4b456-7aac-11e6-9c7f-b993c2130e97.png)

**Comment by IvanSanchez** [comment]
@ormajollila Is this reproducible in 1.0.0?

**Comment by ormajollila** [comment]
@IvanSanchez Nope. On 1.0.0-rc3 this is not a problem.

**Comment by perliedman** [comment]
Looks this is fixed in 1.0. Please add more information if you think this is still an issue.
